src/stories/atoms/Shape/Shape.test.tsx

```diff
@@ -0,0 +1,26 @@
+// Global Imports
+import React from 'react';
+import { composeStories } from '@storybook/testing-react';
+
+// Local Imports
+import { render } from 'utils/test-utils';
+import * as stories from 'stories/atoms/Shape/Shape.stories';
+
+const { Default, Rectangle, Color } = composeStories(stories);
+
+describe('Shape unit tests', () => {
+  it('Should render the Default Shape component', () => {
+    const { getByTestId } = render(<Default />);
+    expect(getByTestId('shape-component-default')).toBeInTheDocument();
+  });
+
+  it('Should render the Rectangle Shape component', () => {
+    const { getByTestId } = render(<Rectangle />);
+    expect(getByTestId('shape-component-rectangle')).toBeInTheDocument();
+  });
+
+  it('Should render the Color Shape component', () => {
+    const { getByTestId } = render(<Color />);
+    expect(getByTestId('shape-component-color')).toBeInTheDocument();
+  });
+});
```

Comment:
Perfect tests. 

I'm passionate about testing so let's please get on a call soon so we can go over these in detail. I'd love to dig deeper on this with you.

Reply:
I'd love to learn more about testing. Right now I can google things to figure it out, but my  understanding of what's possible and the general syntax is lacking.